For_Andrey/main1.1.py

```diff
@@ -0,0 +1,163 @@
+﻿import random  # from random import randint (вызов - randint(2,5))

```

Comment:
1.Remove comments for imports.
2.No Russian language AT ALL!!!
3.Module docstring